docs/trouble-shooting.md

```diff
@@ -0,0 +1,163 @@
+---
+id: trouble-shooting
+title: Trouble Shooting
+---
+
+> “If you know the enemy and know yourself, you need not fear the result of a
+> hundred battles. If you know yourself but not the enemy, for every victory
+> gained you will also suffer a defeat. If you know neither the enemy nor
+> yourself, you will succumb in every battle.”
+>
+> ― Sun Tzu, The Art of War
+
+As developers of Upstream we want you, the user, to have the best experience as
+possible. Also as developers of Upstream, we know that there will be bugs and
+errors. To help consolidate these diametrically opposed pieces of knowledge we
+will try and document any of the errors that we are aware of, and give you
+advice on how you might recover. In the event that this information doesn't
+help, do not hesitate to reach out to [radicle.community](community) — or our
```

Comment:
```suggestion
help, do not hesitate to reach out to [radicle.community][community] — or our
```

The same fix needs to be applied to the other links.

Reply:
https://github.com/radicle-dev/radicle-docs/pull/21/commits/e7b7b2f4163fcb8932e68db1dfc256b6a30699da